packages/cli/defaults/sample-section.html

```diff
@@ -0,0 +1,14 @@
+<!--
+  templateType: section
+-->
+{% dnd_section %}
+  {% dnd_column %}
+    {% dnd_row %}
+      {% dnd_module
+        path='@hubspot/rich_text',
+        html='<h1>Hello world</h1>'
+      %}
+      {% end_dnd_module %}
+    {% end_dnd_row %}
+  {% end_dnd_column %}
+{% end_dnd_section %}
```

Comment:
NIT: missing new line

Reply:
Thanks, resolved in https://github.com/HubSpot/hubspot-cli/pull/546/commits/c95e30d9d7a7e260f4d0af1b8c2efc6a77be658e